prototype/mobile_endpoint/views/receiver.py

```diff
@@ -25,6 +26,7 @@ def couch_receiver(domain):
 @ota_mod.route('/mongo-receiver/<domain>', methods=['POST'])
 @requires_auth
 def mongo_receiver(domain):
+    connect(host=current_app.config.get('MONGO_URI'))
```

Comment:
Initiating the `mongoengine` connection here feels wrong, but didn't spend the time to figure out the "right" place to put this. (Docs seem to suggest at the top of the models file, but `flask.current_app` isn't always known when that is first imported...)